packages/datadog-instrumentations/src/graphql.js

```diff
@@ -211,8 +204,8 @@ function wrapResolve (resolve) {
 
     const field = assertField(context, info, args)
 
-    return callInAsyncScope(resolve, field.asyncResource, this, arguments, context.abortController, (err) => {
-      updateFieldCh.publish({ field, info, err })
+    return callInAsyncScope(resolve, this, arguments, context.abortController, (err) => {
+      updateFieldCh.publish({ field, info, err, ...field.ctx })
```

Comment:
Similar to the above, this doesn't persist the data on the context nor keep the reference.

Reply:
Fixed